addons/point_of_sale/static/src/js/screens.js

```diff
@@ -963,6 +964,42 @@ var ProductScreenWidget = ScreenWidget.extend({
         if (_.size(this.action_buttons)) {
             this.$('.control-buttons').removeClass('oe_hidden');
         }
+
+        this._onKeyDown = function (event) {
```

Comment:
I don't understand why the handlers are defined inline instead of in a main function of the widget?  I would much prefer to put them in the 'Handlers' section, and bind them with the 'events' key for the widget.  Doing it this way will allow the destroy method to properly unbind these handlers.

Note: I am not familiar with the POS, so there might be a good reason for this. In that case, please tell me.